src/document/json/array.ts

```diff
@@ -26,6 +26,7 @@ import { RGATreeList } from './rga_tree_list';
 export class JSONArray extends JSONContainer {
   private elements: RGATreeList;
 
+  /** @hideconstructor */
```

Comment:
After we add `@hideconstructor` tag and run prune-dts script, we can see constructor changed to `private`
<img width="686" alt="Screen Shot 2021-07-03 at 3 46 09 PM" src="https://user-images.githubusercontent.com/4694139/124345772-cb68d080-dc15-11eb-9f0b-a47b19dd2471.png">